"Adding nodes" instructions only work on localhost (#52677)
elastic/elasticsearch: docs/reference/modules/discovery/discovery-settings.asciidoc

Discovery and cluster formation are affected by the following settings:

`discovery.seed_hosts`::
+
--
Provides a list of the addresses of the master-eligible nodes in the cluster.
May also be a single string containing the addresses separated by commas. Each
address has the format `host:port` or `host`. The `host` is either a host name

> to be resolved by DNS, or an IPv4 address, or an IPv6 address. IPv6 addresses

Reviewer: ```suggestion
to be resolved by DNS, an IPv4 address, or an IPv6 address. IPv6 addresses
```

must be enclosed in square brackets. If a host name resolves via DNS to multiple

> addresses then Elasticsearch uses all of them. DNS lookups are subject to

Reviewer: ```suggestion
addresses, {es} uses all of them. DNS lookups are subject to
```

> <<networkaddress-cache-ttl,JVM DNS caching>>. If the `port` is not given then it

Reviewer: ```suggestion
<<networkaddress-cache-ttl,JVM DNS caching>>. If the `port` is not given, it
```

`discovery.seed_hosts`::

    Provides a list of the addresses of the master-eligible nodes in the
    cluster. Each address has the format `host:port` or `host`, where `port`
    defaults to the setting `transport.profiles.default.port` falling back to
    `transport.port` if not set; if neither is set then the default port is
    `9300`. Note that IPv6 addresses must be bracketed. May also be a single

>     string containing all the addresses separated by commas. The default value

Reviewer: > May also be a single string containing all the addresses separated by commas.

It's unclear if this only applies to IPv6 addresses.

Author: Good point, I moved this sentence.

>     is `["127.0.0.1", "[::1]"]`. See <<unicast.hosts>>.

Reviewer: The new info is great, but the user can't quickly scan to see which settings impact the `port`. I think some formatting and whitespace would help.

```
`discovery.seed_hosts`::
+
--
Provides a list of the addresses of the master-eligible nodes in the cluster.
Each address has the format `host:port` or `host`.

The `port` is determined by checking the following settings in order:

. `transport.profiles.default.port`
. `transport.port`

If none of these settings are specified, the port defaults to `9300`.

Note that IPv6 addresses must be bracketed. May also be a single string
containing all the addresses separated by commas. The default value is
`["127.0.0.1", "[::1]"]`. See <<unicast.hosts>>.
--
```

Author: Thanks, yes that's better. I'm never quite sure how to reformat these one-paragraph bits into more richly formatted text, so let's see if my changes worked there...
